client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -202,7 +217,14 @@ const InstalledInOrPricing = ( {
 									<span className="plugins-browser-item__period">{ period }</span>
 								</>
 							) : (
-								translate( 'Free' )
+								<>
+									{ translate( 'Free' ) + ' ' }
```

Comment:
While using `+ ' '` for adding a space works, I think it would be cleaner removing that part from here and adding `margin-left: 5px;` to `.plugins-browser-item__requires-plan-upgrade`